assets/src/scss/layout/navbar/_burger-menu.scss

```diff
@@ -49,220 +49,219 @@
-      }
+  &:not(:last-child) {
+    margin-block-start: $sp-1x;
+  }
 
-      &.collapsed {
-        &::before {
-          transform: rotate(90deg);
-        }
-      }
-    }
+  > a.nav-link, > * > a.nav-link {
+    font-size: 20px;
+    padding: 6px 0;
   }
+}
 
-  .nav-menu-close {
-    display: block;
-    position: absolute;
-    top: 16px;
-    inset-inline-end: $sp-3;
-    border: none;
-    height: 16px;
-    width: 16px;
-    mask: url("../../images/cross.svg") 50% 50%/16px 16px no-repeat;
-    background-color: $grey-80;
-    z-index: 5;
-
-    @supports not (inset-inline-end: $sp-3) {
-      right: $sp-3;
-      left: auto;
+.nav-subitems {
+  padding-inline-start: $sp-2;
+  padding-block-start: $sp-1x;
+  padding-block-end: $sp-1x;
 
-      html[dir="rtl"] & {
-        left: $sp-3;
-        right: auto;
-      }
-    }
-  }
+  .nav-item.active {
+    pointer-events: none;
 
-  .admin-bar & {
-    @include medium-and-less {
-      top: 46px;
-      height: calc(100% - 46px);
-    }
+    a.nav-link {
+      font-weight: bold;
 
-    @media (min-width: 783px) {
-      top: 32px;
-      height: calc(100% - 32px);
+      &:before {
+        border-bottom: none !important;
+      }
     }
   }
 
-  & ~ .burger-menu-overlay {
-    display: none;
-    background: var(--burger-menu-overlay--background, $black);
-    opacity: 0.5;
-    height: 100%;
-    pointer-events: all;
-    position: fixed;
-    top: 0;
-    width: 100vw;
-    z-index: 99;
+  a.nav-link {
+    font-size: 16px;
+    line-height: 19px;
+    font-weight: 400;
+    width: auto;
+    padding: $sp-1x 0;
   }
+}
 
-  @include small-and-less {
-    width: 100vw;
-    inset-inline-start: -100vw;
+.burger-menu-item-wrapper {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
 
-    @supports not (inset-inline-start: -100vw) {
-      transform: translateX(-100vw);
-      transition: transform 0.2s ease-in;
+.collapsable-btn {
+  width: 30px;
+  height: 30px;
+  background: transparent;
+  border: none;
+
+  &::before {
+    content: "";
+    height: 12px;
+    width: 8px;
+    display: inline-block;
+    mask-image: url("../../images/chevron.svg");
+    mask-repeat: no-repeat;
+    mask-size: contain;
+    mask-position: center;
+    background-repeat: no-repeat;
+    background-color: currentColor;
+    transform: rotate(-90deg);
+    transition: transform 0.3s linear;
+    color: $grey-80;
+  }
 
-      html[dir="rtl"] & {
-        transform: translateX(100vw);
-      }
+  &.collapsed {
+    &::before {
+      transform: rotate(90deg);
     }
   }
+}
 
-  @include small-to-large {
-    width: 400px;
-    inset-inline-start: -400px;
-    transition: inset-inline-start 0.2s ease-in;
-
-    @supports not (inset-inline-start: -400px) {
-      transform: translateX(-400px);
-      transition: transform 0.2s ease-in;
-
-      html[dir="rtl"] & {
-        transform: translateX(400px);
-      }
+.nav-menu-close {
```

Comment:
If you move outside the `.burger-menu` class, you're converting this as a general class and I think it would be nice (only in this case) to set the specificity. What do you think?

Reply:
We only use this class [in 1 place in 1 Twig template](https://github.com/search?q=org%3Agreenpeace+%22nav-menu-close%22&type=code).

And if you think about it, why would these styles only make sense if the close button is inside a burger menu? What if we put the same close button on something that is not a burger menu item.

If somebody would decide to in our code start using the class name in another place, they'll immediately see that style applied. If it doesn't make sense for that item, you can just use another class. If the styles already work on the new element, you don't have to adjust any CSS. Lower specificity = CSS that you can actually reuse.